release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -185,9 +191,9 @@ With this release, you can access the NMstate Operator and resources such as the
 === Machine Config Operator
 
 [id="ocp-4-14-mco-ca-distribution"]
-==== Handling of registry certificate authorities 
+==== Handling of registry certificate authorities
 
-The Machine Config Operator now handles distributing certificate authorities for image registries. This change does not affect end users. 
+The Machine Config Operator now handles distributing certificate authorities for image registries. This change does not affect end users.
```

Comment:
```suggestion
The Machine Config Operator now handles the distribution of certificate authorities for image registries. This change does not affect end users.
```

Reply:
@agantony I believe this content is not a part of the changes. It's appearing only because of trailing spaces removal.